### ISSUE TITLE
Set status of published page to unpublished

### ISSUE THREAD

**Issue by andreash** [jgarber/radiant-chronicle-extension#30]
You created a great extension, I like the idea and the concept. I have noticed a few things and would be glad if they can be resolved. Here is one.

I have pages which are in a published state and a new release is in an unpublished state. I decided to take the published page from the net and wait until the new version is ready to be published. I could not find a way to do that. 

A possible solution might be to be able to set the status of the published page to "draft" or "review" and thereby make it "just one of the old versions".

**Comment by jgarber** [comment]
Do you want to take the page offline?  If so, make it hidden.  If not, please explain in more detail.

**Comment by andreash** [comment]
No, I wanted the unpublished version still to be visible in dev mode. I could not find a way to tell the system "hide the published version but keep the unpublished version online in dev mode"

**Comment by jgarber** [comment]
I think I understand.  You'll want to save a version as hidden and then save it as draft/reviewed. Then it will be hidden in live mode and visible in dev.

**Comment by andreash** [comment]
Almost - the understanding of the result is correct. I have created a page and published it. Later I decide that the content became invalid and started to change the content and put the page to "draft" status so that it was visible in dev. But the published version is still there and there seems no way to "hide" it.

The workaround I found was to create a new published version first which uses a layout with a redirect in it and then continue to develop the page contents in draft status. But that a little cumbersome.
